extensions/amp-story-panning-media/README.md

```diff
@@ -0,0 +1,42 @@
+# Interactive components
```

Comment:
Components don't typically have README.md files - this one should be removed. I've suggested how to link to the demos in their current state in amp-story-panning-media.md